src/main/java/se/michaelthelin/spotify/SpotifyApi.java

```diff
@@ -1358,6 +1358,15 @@ public AddItemToUsersPlaybackQueueRequest.Builder addItemToUsersPlaybackQueue(St
       .uri(uri);
   }
 
+  /**
+   * Receive all tracks from the user's current playback queue.
+   *
+   */
+  public GetListOfTracksFromUsersPlaybackQueueRequest.Builder getListOfTracksFromUsersPlaybackQueue() {
```

Comment:
All endpoints are named as given in the api reference or at least as they were given at some point. Looking at the api reference right now, this would need to be called `GetTheUsersQueueRequest`, right?